# Patches for microb-engine
# Oleg Romashin <oleg.romashin@nokia.com>
# DP: Building libxul when enabled system nss
# NOBUGFIXME

Index: mozilla/toolkit/library/libxul-rules.mk
===================================================================
--- mozilla.orig/toolkit/library/libxul-rules.mk
+++ mozilla/toolkit/library/libxul-rules.mk
@@ -37,18 +37,23 @@
 # ***** END LICENSE BLOCK *****
 
 EXTRA_DSO_LDOPTS += \
 	$(MOZ_FIX_LINK_PATHS) \
 	$(LIBS_DIR) \
 	$(JPEG_LIBS) \
 	$(PNG_LIBS) \
 	$(MOZ_JS_LIBS) \
+	$(NULL)
+
+ifdef MOZ_PSM
+EXTRA_DSO_LDOPTS += \
 	$(NSS_LIBS) \
 	$(NULL)
+endif
 
 ifdef MOZ_NATIVE_ZLIB
 EXTRA_DSO_LDOPTS += $(ZLIB_LIBS)
 else
 EXTRA_DSO_LDOPTS += $(MOZ_ZLIB_LIBS)
 endif
 
 # need widget/src/windows for resource.h (included from widget.rc)
